.circleci/config.yml

```diff
@@ -112,8 +112,12 @@ jobs:
       # See options explanation here: https://docs.codecov.com/docs/codecov-uploader
       - codecov/upload:
           flags: ios-tests
-          upload_name: Coverage Report for iOS Tests
-          xtra_args: -c -v --xc --xp iosresults.xcresult
+          upload_name: Coverage report for iOS functional tests
+          xtra_args: -c -v --xc --xp iosFunctionalResults.xcresult
+      - codecov/upload:
+          flags: ios-tests
+          upload_name: Coverage report for iOS unit tests
+          xtra_args: -c -v --xc --xp iosUnitResults.xcresult
```

Comment:
nit: can we have unit tests coverage first, then functional  (since functional is more generic than unit)